inference-engine/tests/ie_test_utils/unit_test_utils/mocks/mock_not_empty_icnn_network.hpp

```diff
@@ -43,6 +43,10 @@ class MockNotEmptyICNNNetwork final : public ICNNNetwork {
     MOCK_METHOD(StatusCode, reshape, (const ICNNNetwork::InputShapes &, ResponseDesc *), (noexcept));
     MOCK_METHOD(StatusCode, serialize,
         (const std::string &, const std::string &, InferenceEngine::ResponseDesc*), (const, noexcept));
+    MOCK_METHOD(StatusCode, serialize,
+        (std::ostream &, std::ostream &, InferenceEngine::ResponseDesc*), (const, noexcept));
+    MOCK_METHOD(StatusCode, serialize,
+        (std::ostream &, Blob::Ptr &, InferenceEngine::ResponseDesc*), (const, noexcept));
```

Comment:
We need add more tests with model reading, serialization and then simple checks.